core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -425,6 +445,9 @@ UNWIND(	.cantunwind)
 
 	bl	plat_cpu_reset_late
 
+    /* this core just comes out of reset, discard undefined l1 contents */
+	bl	arm_cl1_d_invbysetway
```

Comment:
>  This is safe because its "cache invalidate" on L1, other cores would not respond to this op.

ok. thanks for the details.
looks clear.

> And I wonder why this is not required (...)

on armv8, cache were cleaned by el3.
on armv7, it is likely that for those legacy platforms an earlier bootloader invalidates caches.

> I'll put it inside CFG_TEE_VXWORKS_SUPPORT to reflect this is due to VxWorks's special boot scenario.

Since introducing CFG_TEE_VXWORKS_SUPPORT , it could be nice to have this dependency info within optee.
Could you elaborate some make command using `$(call cfg-depends-one/all,`... to detect the dependency, and integrate it into mk/config.mk ?